linera-views/src/backends/scylla_db.rs

```diff
@@ -715,6 +718,33 @@ impl AdminKeyValueStore for ScyllaDbStoreInternal {
         Ok(namespaces)
     }
 
+    async fn list_root_keys(
+        config: &Self::Config,
+        namespace: &str,
+    ) -> Result<Vec<Vec<u8>>, ScyllaDbStoreInternalError> {
+        Self::check_namespace(namespace)?;
+        let session = SessionBuilder::new()
+            .known_node(config.uri.as_str())
+            .build()
+            .boxed()
+            .await?;
+        // Create a table if it doesn't exist
+        // The schema appears too complicated for non-trivial reasons.
+        // See TODO(#1069).
```

Comment:
#1069 is already closed. Do we need to the TODO?

Reply:
Yes, thank you.